tests/core/test_oauth_clients.py

```diff
@@ -1,10 +1,12 @@
 from flask import json
+import pytest
 from newsroom.tests.users import test_login_succeeds_for_admin
 from superdesk import get_resource_service
 from newsroom.auth_server.auth import JWTAuth
 import base64
 
 
+@pytest.mark.skip(reason="Pending to figure out some issue with OAuth2")
```

Comment:
I would like to have a separate ticket to take care of this part. Need to understand the whole logic to provide a proper fix.

Reply:
We have SDESK-7284 and NHUB-511 specifically to fix breaking changes. We can fix this as part of those tickets